showermaxgen.py

```diff
@@ -156,28 +149,21 @@
 out+="\n\t\t<rotation name=\"rot_logic_mirror_box\" x=\"0\" y=\"0\" z=\"0\"/>"
 out+="\n\t</union>\n"
 
-
 out+="\t<union name=\"solid_logic_mirror_box_union\">"
```

Comment:
I think `solid_logic_mirror_box_union` needs another union to be big enough for the front and back plates, and `solid_logic_mirror_box` needs to be shifted by half a tungsten layer thickness.